projects/generate_confs/parser.ypp

```diff
@@ -407,6 +409,8 @@ nhits: TK_NHITS '=' int_numb {$$=$3;};
 
 noise_type: TK_NOISE_TYPE '=' TK_RND_T {$$=$3;};
 
+method:flag TK_METHOD_FLAG '=' int_numb {$$=$3;};
```

Comment:
shouldn't method:flag become method_flag? anyway I would call it simply "method"